docs/source/internals.rst

```diff
@@ -411,16 +411,88 @@ Conditional events
 
 ::
 
-    - [API]           - only for corresponding RP API calls
-    - [CFG]           - only for some RP configurations
-      - [CFG-R]       - only for some bootstrapping configurations
-      - [CFG-DVM]     - only for launch methods which use a DVM
-    - [Task]          - only for some Task descriptions
-      - [Task-DS]     - only for tasks specifying data staging directives
-      - [Task-PRE]    - only for tasks specifying pre-exec directives
-      - [Task-POST]   - only for tasks specifying post-exec directives
-    - [PILOT]         - only for certain pilot
-    - [APP]           - only for applications writing compatible profiles
-    - [RUNTIME]       - only on  certain runtime decisions and system configuration
-    - [INTERNAL]      - only for certain internal states
+   - [API]           - only for corresponding RP API calls
+   - [CFG]           - only for some RP configurations
+     - [CFG-R]       - only for some bootstrapping configurations
+     - [CFG-DVM]     - only for launch methods which use a DVM
+   - [Task]          - only for some Task descriptions
+     - [Task-DS]     - only for tasks specifying data staging directives
+     - [Task-PRE]    - only for tasks specifying pre-exec directives
+     - [Task-POST]   - only for tasks specifying post-exec directives
+   - [PILOT]         - only for certain pilot
+   - [APP]           - only for applications writing compatible profiles
+   - [RUNTIME]       - only on  certain runtime decisions and system configuration
+   - [INTERNAL]      - only for certain internal states
+
+
+Implementing Executors
+======================
+
+Agent executors are at the heart of RADICAL-Pilot: they are responsible for
+executing tasks on the target resources. The executor is a Python class that
+implements the `radical.pilot.agent.executing.Executor` interface.
```

Comment:
since this is a RST file, nee to use double back-quoting (the same for other code words)
```
``radical.pilot.agent.executing.Executor``
```

Reply:
Thanks, fixed.